### DISCUSSION TITLE
Sub-directory redirects to 404 page at root instead of index.html inside

### DISCUSSION THREAD

**Discussion by paramt** [community/community#22296] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://www.param.me" rel="nofollow noopener">My site</a> has a 404 page that just redirects to the index.html at root (e.g. <a href="http://www.param.me/foo" rel="nofollow noopener">www.param.me/foo</a> redirects to <a href="http://www.param.me" rel="nofollow noopener">www.param.me</a> since it doesn’t exist) That works as expected…</p>
<p>However, I have a sub-directory called /old with an index.html inside. When I navigate to <a href="https://www.param.me/old" rel="nofollow noopener">www.param.me/old</a>, instead of redirecting to <a href="http://www.param.me/old/index.html" rel="nofollow noopener">www.param.me/old/index.html</a>, it redirects to the root 404 page</p>
<p><a href="https://github.com/paramt/paramt.github.io" rel="nofollow noopener">Github repo</a></p>
<p>EDIT: I checked back 4 hours later and it seems to work now… I guess Github Pages just needed some time to recognize /old as a valid sub-directory</p>

**Comment by RDIL** [comment]
<p>Yeah, I’ve noticed this also. </p>
<p>For example:</p>
<p><a href="https://rdil.rocks/hdgfuidhfiguhsuig" rel="nofollow noopener">https://rdil.rocks/hdgfuidhfiguhsuig</a></p>
<p>vs</p>
<p><a href="https://rdil.rocks/gfsjdg/ffigisdjg" rel="nofollow noopener">https://rdil.rocks/gfsjdg/ffigisdjg</a></p>

**Comment by paramt** [comment, marked as answer]
<p>I checked back 4 hours later and it seems to work now… I guess Github Pages just needed some time to recognize /old as a valid sub-directory</p>
